wallets/ethereum-wallet-mock/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './ethereum-wallet-mock'
+export * from './EthereumWalletMock'
```

Comment:
also.. in future, is it going to accept different network, like solana? then maybe it should be named differently?

Reply:
in tests, we're calling it `walletMock` which suggests it will accept different networks in future

Reply:
Different chains require different implementation - even for mock